README.md

```diff
@@ -18,7 +18,7 @@ A distributed ledger for ecology built on top of the
 ## Getting Started
 
 
-See [Getting Started](docs/getting-started.md) for instructions how to install and run the blockchain.
+See [Getting Started](docs/getting-started/README.md) for instructions how to install and run the blockchain.
```

Comment:
Maybe this should point to https://docs.regen.network/getting-started/ since that's where the links will all function as expected?

Reply:
@clevinson I think pointing to the URL you are suggesting makes a lot more sense. I wasn't aware that existed and was just trying to make this functional. Happy to replace the inter-GitHub link with the link to full docs domain.

Reply:
Looks like @ryanchristo is happy to accept this change as is, but create a follow-up PR to sweep this more fully and link out to the official docs.